kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/prometheus/PrometheusIntegrationTest.java

```diff
@@ -211,7 +212,54 @@ public void testPrometheusJobMetrics() throws Exception {
 //              "kie_server_job_duration_seconds_sum{container_id=\"\",command_name=\"" + PRINT_OUT_COMMAND + "\",}"
           );
     }
-
+    
+	@Test
+	@Category(JEEOnly.class) // Executor in kie-server-integ-tests-all is using JMS for execution. Skipping test for non JEE containers as they don't have JMS.
+	public void testPrometheusJobErrorMetrics() throws Exception {
+		JobRequestInstance jobRequestErrorInstanceNow = createJobRequestExecutionErrorInstance();
+		Long jobId = jobServicesClient.scheduleRequest(jobRequestErrorInstanceNow);
+		KieServerSynchronization.waitForJobToFinish(jobServicesClient, jobId);
+
+		assertThat(getMetrics()).contains(
+				"kie_server_job_in_retry_total{container_id=\"\",failed=\"true\",command_name=\""
+						+ JOB_EXECUTION_ERROR_COMMAND + "\",}",
```

Comment:
You could check that metric is 0.0
```suggestion
				"kie_server_job_in_retry_total{container_id=\"\",failed=\"true\",command_name=\""
						+ JOB_EXECUTION_ERROR_COMMAND + "\",} 0.0",
```